omnibus/config/projects/chef.rb

```diff
@@ -59,6 +71,12 @@
 dependency "shebang-cleanup"
 dependency "version-manifest"
 dependency "openssl-customization"
+dependency "rb-readline"
```

Comment:
You only need this on windows right?  Other platforms use libedit or whatever else we compile in.